project/datamodules/text/hf_text.py

```diff
@@ -0,0 +1,238 @@
+from __future__ import annotations
+
+import shutil
+from logging import getLogger
+from pathlib import Path
+
+import numpy as np
+import torch
+from datasets import DatasetDict, load_dataset
+from lightning import LightningDataModule
+from torch.utils.data import DataLoader
+from transformers import (
+    AutoTokenizer,
+)
+
+from project.utils.env_vars import REPO_ROOTDIR, SCRATCH, SLURM_TMPDIR
+from project.utils.utils import get_task_info
+
+torch.set_num_threads(
```

Comment:
Also: If we do this here, it will always be run when importing the project, not just when using this datamodule.
Did this help make the test run faster?

Reply:
fixed in 568b537c2bd0543066cf3b9ec260fc778c8ceed3